### DISCUSSION TITLE
Xcode 14 previews sees the / operator as the start of a regex literal

### DISCUSSION THREAD

**Discussion by lukesutton** [pointfreeco/swift-case-paths#80] [Q&A]
My projects using Case Paths are compiling and running fine. However Xcode previews believe that the `/` operator is the start of the new RegEx literal, and throws an error about it being unterminated.

So, this is not an issue in Case Paths per se, but I wonder, is there some work around? I'd happily use a function call in the meantime, but unfortunately I've been stuck trying to figure out all the overloads for `/` and how I might wrap them with a function instead.

**Comment by stephencelis** [comment]
Hi @lukesutton! This is a known bug in the Xcode beta. According to Apple if you add a comment to the end of a line with a case path, that should get previews compiling.

```diff
-.pullback(/Enum.case)
+.pullback(/Enum.case)  // NB: Comment workaround for Xcode 14 beta 1 previews
```

Also check out these discussions around the matter:

- https://github.com/pointfreeco/swift-case-paths/discussions/75
- https://github.com/pointfreeco/swift-case-paths/discussions/77

I'm going to convert this to a discussion, as well, since it's an Apple bug and not a bug with our library.
